beater/mysqlbeat.go

```diff
@@ -79,49 +79,42 @@ func New(b *beat.Beat, cfg *common.Config) (beat.Beater, error) {
 		config: config,
 	}
 
-	// Handle password decryption and save in the bt
-	if bt.config.EncryptedPassword != "" {
-		aesCipher, err := aes.NewCipher([]byte(secret))
-		if err != nil {
-			return nil, err
-		}
-		cfbDecrypter := cipher.NewCFBDecrypter(aesCipher, commonIV)
-		chiperText, err := hex.DecodeString(bt.config.EncryptedPassword)
-		if err != nil {
-			return nil, err
-		}
-		plaintextCopy := make([]byte, len(chiperText))
-		cfbDecrypter.XORKeyStream(plaintextCopy, chiperText)
-		bt.password = string(plaintextCopy)
-	} else {
-		bt.password = bt.config.Password
-	}
-
 	// init the oldValues and oldValuesAge array
 	bt.oldValues = common.MapStr{"mysqlbeat": "init"}
 	bt.oldValuesAge = common.MapStr{"mysqlbeat": "init"}
 
-	// Save config values to the bt
-	bt.hostname = bt.config.Hostname
-	bt.port = bt.config.Port
-	bt.username = bt.config.Username
-	bt.queries = bt.config.Queries
-	bt.queryTypes = bt.config.QueryTypes
 	bt.deltaWildcard = bt.config.DeltaWildcard
 	bt.deltaKeyWildcard = bt.config.DeltaKeyWildcard
 
 	safeQueries := true
 
-	logp.Info("Total # of queries to execute: %d", len(bt.queries))
-	for index, queryStr := range bt.queries {
+	for server, server_params := range bt.config.Servers {
+		// Decrypt passwords for servers
+		if val, ok := server_params["EncryptedPassword"]; ok {
```

Comment:
`val` is a bad name. Something like `encryptedPassword` would be a better choice. (names should have meaning)